src/ildasm/dasm.cpp

```diff
@@ -7873,12 +7877,13 @@ BOOL DumpFile()
                     {   // initialized data
                         szptr+=sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s%8.8X = %s (",szTls,fromPtr,KEYWORD("bytearray"));
                         printLine(g_pFile,szString);
-                        sprintf_s(szString,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
+                        szptr = szString;
+                        sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
```

Comment:
The code was ok before although not really clear. We printed the indentation to szString and then inside DumpHexbytes we would check how much was printed so far and start adding from there.

With my fix, this is useless to check for the length so I suggest this change in DumpHexBytes:

```
diff --git a/src/ildasm/dasm.cpp b/src/ildasm/dasm.cpp
index e4f0025..bb5f5e5 100644
--- a/src/ildasm/dasm.cpp
+++ b/src/ildasm/dasm.cpp
@@ -6509,10 +6509,9 @@ void DumpStatistics(IMAGE_COR20_HEADER *CORHeader, void* GUICookie)
 #pragma warning(pop)
 #endif

-void DumpHexbytes(__inout __nullterminated char* szString,BYTE *pb, DWORD fromPtr, DWORD toPtr, DWORD limPtr)
+void DumpHexbytes(__inout __nullterminated char* szptr,BYTE *pb, DWORD fromPtr, DWORD toPtr, DWORD limPtr)
 {
     char sz[256];
-    char* szptr = &szString[strlen(szString)];
```

In any case the indentation is good.